.github/workflows/ci.yml

```diff
@@ -40,7 +40,27 @@ jobs:
     # Build package
     - name: build
       run: npm run build
-    
+
+  test:
+    name: Test & Coverage
+    runs-on: ubuntu-latest
+    steps:
+    # Check out repo
+    - uses: actions/checkout@master
+
+    # Cache node modules
+    - name: Cache node_modules
+      id: cache-modules
+      uses: actions/cache@v1
+      with:
+        path: node_modules
+        key: ${{ runner.OS }}-build-${{ hashFiles('package.json') }}
+
+    # Install dependencies - use cache if possible
+    - name: npm install
+      if: steps.cache-modules.outputs.cache-hit != 'true'
+      run: npm install
```

Comment:
https://docs.npmjs.com/cli/v6/commands/npm-ci
https://vxcompany.com/2019/02/19/version-management-of-npm-packages/

We should consider using `npm ci` instead, maybe